Upgrade: add v1.1.1/v1.1.2 -> v1.1.3 path (#492)
harvester/docs: docs/upgrade/v1-1-1-to-v1-1-3.md

</head>

## General information

An **Upgrade** button appears on the Dashboard screen whenever a new Harvester version that you can upgrade to becomes available. For more information, see [Start an upgrade](./automatic.md#start-an-upgrade).

For air-gapped environments, see [Prepare an air-gapped upgrade](./automatic.md#prepare-an-air-gapped-upgrade).

## Known Issues

---

### 1. The upgrade process is stuck when pre-draining a node. (Case 1)

Starting from v1.1.0, Harvester waits for all volumes to become healthy before upgrading a node (for clusters with three or more nodes). 

When this issue occurs, you can check the health of the affected volumes on the [embedded Longhorn UI](../troubleshooting/harvester.md#access-embedded-rancher-and-longhorn-dashboards). You can also check the pre-drain job logs. For more troubleshooting information, see [Phase 4: Upgrade nodes](./troubleshooting.md#phase-4-upgrade-nodes).

---

### 1. The upgrade process is stuck when pre-draining a node. (Case 1)

An upgrade is stuck, as shown in the screenshot below:

![](/img/v1.2/upgrade/known_issues/3216-stuck-pre-drain.png)

> Harvester is unable to proceed with the upgrade and the status of at least two node are `SchedulingDisabled`.

Reviewer: ```suggestion
Harvester is unable to proceed with the upgrade and the status of two or more nodes is `SchedulingDisabled`.
```
